WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -646,6 +645,8 @@ extension WordPressAppDelegate {
 #if JETPACK
         case is MigrationNavigationController:
             return "Jetpack Migration View"
+        case is MigrationLoadWordPressViewController:
+            return "Jetpack Migration Load WordPress View"
```

Comment:
Perhaps for a future note, but the root cause is the premature initialization of `WPTabBarController`. While adding the switch cases for possible root view controllers would solve the issue, it is a time bomb waiting to happen when a new root view controller is added.

I think this can be solved by not calling the `RootViewCoordinator.sharedPresenter` in the default case and instead adding a specific case for `WPTabBarController`. The default case should return nil instead. This way, the analytics value would be correct when there's an unhandled root view controller (i.e., returning `nil` instead of the default tab of the `WPTabBarController`, which is not visible to the user).

However, this should probably be implemented in a separate PR to avoid scope-creeping this PR. 😅 I can look into it this week.